drivers/FlashIAP.h

```diff
@@ -40,6 +40,13 @@ namespace mbed {
  */
 class FlashIAP : private NonCopyable<FlashIAP> {
 public:
+
+/**
```

Comment:
this should be still removed (you can rebase to clean up the history to have only 1 commit here)

Reply:
Uhhh...